example/index.js

```diff
@@ -0,0 +1,63 @@
+// example/index.js
+import { UltravoxSession } from '../dist/esm/index.js';
+
+class UltravoxExample {
+  constructor() {
+    this.uvSession = new UltravoxSession();
+    this.setupEventListeners();
+  }
+
+  appendUpdate(target, message) {
+    const updateTarget = document.getElementById(target);
+
+    if (target === 'callTranscript') {
+      let transcriptText = '';
+      message.forEach((transcript, index) => {
+        if (transcript.speaker === 'agent') {
+          transcriptText += '<p>' + transcript.speaker + ': ' + transcript.text + '</p>';
+        }
+      });
+      updateTarget.innerHTML = transcriptText;
+      updateTarget.scrollTop = updateTarget.scrollHeight;
+    } else {
+      updateTarget.innerHTML = `<p>Call Status: ${message}</p>`;
+    }
+  }
+
+  setupEventListeners() {
```

Comment:
nit: "set up" is two words when used as a verb

```suggestion
    this.setUpEventListeners();
  }

  appendUpdate(target, message) {
    const updateTarget = document.getElementById(target);

    if (target === 'callTranscript') {
      let transcriptText = '';
      message.forEach((transcript, index) => {
        if (transcript.speaker === 'agent') {
          transcriptText += '<p>' + transcript.speaker + ': ' + transcript.text + '</p>';
        }
      });
      updateTarget.innerHTML = transcriptText;
      updateTarget.scrollTop = updateTarget.scrollHeight;
    } else {
      updateTarget.innerHTML = `<p>Call Status: ${message}</p>`;
    }
  }

  setUpEventListeners() {
```